### PR TITLE
Add Vagrantfile, expand README with setup steps, add homestead defaults

### PR DESCRIPTION
**Pull request by bpkennedy**
So, between reading the [Laravel/Homestead documentation](https://laravel.com/docs/8.x/homestead#first-steps), #per-project homestead examples, and some trial and error to ensure that it worked from a new environment machine (in my case I did not even have PHP installed), I think I have some basic setup steps baked into the repository now.

My hope is that this will be helpful for those wanting to contribute to this Laravel API project but may not have much PHP experience.  Also Vagrant can be new (as it was to me), and even managing VM's in something like VirtualBox.  So, maybe this can more sweepingly wrap things up into a "Getting Started" of our own, while allowing those with more knowledge to still modify their setup however they like.

Sound ok?  Thoughts?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+When contributing to this repository, please first discuss the change you wish to make via issue,
+[PBBG Discord](https://discord.gg/tr6d7d4), or any other method with the owners of this repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. We *highly* encourage [short, concise git commit messages](https://chris.beams.io/posts/git-commit/).
+2. Ensure any eslint (including warnings) and cypress tests pass locally before creating your Pull Request.
+3. After Pull Request creation, your branch must pass the eslint and UAT tests that run automatically.
```
Comment (foohonpie) [inline CONTRIBUTING.md:12]: needs adjustment to our BE equivalent

File: README.md [hunk 2]
```diff
@@ -1,24 +1,104 @@
...
 
-[![Build Status](https://travis-ci.org/laravel/lumen-framework.svg)](https://travis-ci.org/laravel/lumen-framework)
-[![Total Downloads](https://poser.pugx.org/laravel/lumen-framework/d/total.svg)](https://packagist.org/packages/laravel/lumen-framework)
-[![Latest Stable Version](https://poser.pugx.org/laravel/lumen-framework/v/stable.svg)](https://packagist.org/packages/laravel/lumen-framework)
-[![License](https://poser.pugx.org/laravel/lumen-framework/license.svg)](https://packagist.org/packages/laravel/lumen-framework)
+# api.pbbg.com
+The API for Persistent Browser-Based Games.
 
-Laravel Lumen is a stunningly fast PHP micro-framework for building web applications with expressive, elegant syntax. We believe development must be an enjoyable, creative experience to be truly fulfilling. Lumen attempts to take the pain out of development by easing common tasks used in the majority of web projects, such as routing, database abstraction, queueing, and caching.
+## Prerequisites for host machine
```
Comment (foohonpie) [inline README.md:6]: Let's change this to "recommended" or similar phrasing t; there are other dev setups, including non-vagrant setups, that would work fine.

File: aliases [hunk 3]
```diff
@@ -0,0 +1,283 @@
+alias ..="cd .."
```
Comment (foohonpie) [inline aliases:1]: What's the source of these aliases in general?  There's some stuff here that isn't relevant to our needs.

**Review by foohonpie — CHANGES_REQUESTED** [review]
This is awesome!

I left a few comments/questions.  Thanks.

File: aliases (hunk 3 above)
Comment (bpkennedy) [reply aliases:1]: Yea, that was one of the auto-generated files that was made when I did the per-project installation of homestead.

I had run `composer require laravel/homestead --dev` And then after I did `php vendor/bin/homestead make` and aliases and after.sh were part of the output of that.  I'm not sure if they are necessary?  *shrugs*. But I'm a fan of removing stuff that isn't useful so I'll try removing it and see if all builds fine?

FYI, I was kinda following this article https://martinbean.dev/blog/2016/10/06/using-homestead-on-a-per-project-basis/ along with the homestead docs.

**Review by foohonpie — APPROVED** [review]
Looks good!  Thank you.

**Pull request merged**